src/Base/AMReX.cpp

```diff
@@ -21,6 +25,51 @@ void init_AMReX(py::module& m)
                     py::return_value_policy::reference)
         ;
 
+    py::class_<Config>(m, "Config")
```

Comment:
Shall we move this over to a `Config.cpp` file?

Can act as a py-counterpart to `AMReX_Config.H`.

Reply:
I thought about that... but `Verbose(), OutStream(), Version()` are actually defined in `AMReX.H` so it felt better to keep it along with that.

Reply:
May be we should name that class `Settings` instead if `Config` is confusing... that still doesn't play nice with `OutStream` and `Print` though :(

Reply:
Settings is a great idea, too.

I often called this `get(Build)Variant(s)` in the past, but that's probably not too obvious either.

Reply:
Slack: staying with *config* for now since its similar in other python packages (e.g. mpl)